packages/data-store/src/types.ts

```diff
@@ -44,7 +44,7 @@ export type DataStoreConfigs = {
 
   // Typeorm supported multiple RDBMS types https://typeorm.io/data-source-options#common-data-source-options
   // We only tested sqlite and react-native sqlite so far
```

Comment:
should we update this comment? Or is sqljs just another sqlite wrapper?

Reply:
good point, sqljs should be added

Reply:
let me push an update here